invenio_records_marc21/resources/serializers/ui/schema.py

```diff
@@ -35,7 +35,17 @@ class Marc21UISchema(Marc21Schema):
 
     id = SanitizedUnicode(data_key="id", attribute="id")
 
-    access_status = AccessStatusField(attribute="access")
```

Comment:
question: is this not necessary anymore?

Reply:
No is not necessary in the UI. The access field should stay the same to work with the record access component.